test/parsing/errors/multipleClassKinds-detailed.good

```diff
@@ -0,0 +1,8 @@
+─── error in multipleClassKinds.chpl:2 [MultipleClassKinds] ───
+  Type expression uses multiple class kinds: owned owned.
```

Comment:
Is "owned owned" how the output was before? I feel like saying "the first is 'owned', the second is 'shared'" would make this read better (but a counterpoint might be that it dilutes the information in the error messages / makes it harder to tell at a glance).

Reply:
Yes, that's what was there before. I see your point but also would take that counterpoint. How about `Type expression uses multiple class kinds ("owned" and "owned")`?

Reply:
I like that a lot.